tests/pipeline/nodes/model/mtcnnv1/test_mtcnn.py

```diff
@@ -45,6 +45,15 @@ def mtcnn_config():
     return node_config
 
 
+@pytest.fixture
+def model_dir(mtcnn_config):
+    return (
+        mtcnn_config["root"].parent
+        / "peekingduck_weights"
+        / mtcnn_config["weights"]["model_subdir"]
+    )
```

Comment:
Not critical for this PR but something to consider for the future. If we can standardize the structure of `weights` enough, e.g., use a dictionary even if the model only has 1 weights file/model type like
```yaml
model_type: default
weights_parent_dir: null
weights: {
    model_subdir: mtcnn,
    blob_file: mtcnn.zip,
    model_file: {
        default: mtcnn.pb
    }
}
```
We might be able to move this fixture to conftest.py and potentially make it easier to consolidate weights loading to a single helper class

Reply:
Noted, we can KIV for future.